src/sass/pages/home.scss

```diff
@@ -43,6 +43,7 @@
       border: 0;
       white-space: normal;
       font-size:18px;
+      -webkit-appearance:none;
```

Comment:
This is how it looks like at the moment:
(iPhone 6 plus chrome)
![photo](https://cloud.githubusercontent.com/assets/4541182/9517365/30f462b2-4ce2-11e5-85c7-dec0740e232a.jpg)